src/main/js/samples.js

```diff
@@ -46,36 +46,82 @@ samples.push({
     name: 'hello',
     title: 'Hello World',
     script:
-        "node {\n" +
-        "   echo 'Hello World'\n" +
+        "pipeline {\n" +
+        "   agent any\n" +
+        "\n" +
+        "   stages {\n" +
+        "      stage('Hello') {\n" +
+        "         steps {\n" +
+        "            echo 'Hello World'\n" +
+        "         }\n" +
+        "      }\n" +
+        "   }\n" +
         "}"
-}); 
+});
 
 samples.push({
     name: 'github-maven',
     title: 'GitHub + Maven',
     script:
-        "node {\n" +
-        "   def mvnHome\n" +
-        "   stage('Preparation') { // for display purposes\n" +
-        "      // Get some code from a GitHub repository\n" +
-        "      git 'https://github.com/jglick/simple-maven-project-with-tests.git'\n" +
-        "      // Get the Maven tool.\n" +
-        "      // ** NOTE: This 'M3' Maven tool must be configured\n" +
-        "      // **       in the global configuration.           \n" +
-        "      mvnHome = tool 'M3'\n" +
+        "pipeline {\n" +
+        "   agent any\n" +
+        "\n" +
+        "   tools {\n" +
+        "      // Install the Maven version configured as \"M3\" and add it to the path.\n" +
+        "      maven \"M3\"\n" +
         "   }\n" +
-        "   stage('Build') {\n" +
-        "      // Run the maven build\n" +
-        "      if (isUnix()) {\n" +
-        "         sh \"'${mvnHome}/bin/mvn' -Dmaven.test.failure.ignore clean package\"\n" +
-        "      } else {\n" +
-        "         bat(/\"${mvnHome}\\bin\\mvn\" -Dmaven.test.failure.ignore clean package/)\n" +
+        "\n" +
+        "   stages {\n" +
+        "      stage('Build') {\n" +
+        "         steps {\n" +
+        "            // Get some code from a GitHub repository\n" +
+        "            git 'https://github.com/jglick/simple-maven-project-with-tests.git'\n" +
+        "\n" +
+        "            // Run Maven on a Unix agent.\n" +
+        "            sh \"mvn -Dmaven.test.failure.ignore=true clean package\"\n" +
+        "\n" +
+        "            // To run Maven on a Windows agent, use\n" +
+        "            // bat \"mvn -Dmaven.test.failure.ignore=true clean package\"\n" +
```

Comment:
Well this is pretty unfortunate.

Reply:
Got me an idea :)
https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/249